src/main/java/com/ing/data/cassandra/jdbc/utils/ArrayImpl.java

```diff
@@ -0,0 +1,76 @@
+package com.ing.data.cassandra.jdbc.utils;
+
+import java.sql.Array;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.Map;
+
+/**
+ * @author Liudmila Kornilova
+ **/
+public class ArrayImpl implements Array {
+    private Object[] array;
+
+    public ArrayImpl(Object[] array) {
+        this.array = array;
+    }
+
+    @Override
+    public String getBaseTypeName() throws SQLException {
+        throw new SQLFeatureNotSupportedException();
+    }
+
+    @Override
+    public int getBaseType() throws SQLException {
+        throw new SQLFeatureNotSupportedException();
+    }
+
+    @Override
+    public Object[] getArray() throws SQLException {
+        if (array == null) throw new SQLException("Array was freed");
```

Comment:
To be compliant with the Checkstyle rules, the `if` block must always use braces.